NEWS.md

```diff
@@ -3,6 +3,10 @@ NEWS
 
 See the changelog for a full list of changes.
 
+1.0.2 Bugfix release
+--------------------
+No changes needed.
```

Comment:
force= parameter in message lib?

Reply:
This file is for users, not for developers.